fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/TimeRoundSeries.java

```diff
@@ -91,7 +92,7 @@ private static LocalDateTime getDateCeilOrFloor(DATE tag, LocalDateTime date, in
                 break;
             }
             default: {
-                return null;
+                throw new AnalysisException("Unexpected tag: " + tag);
```

Comment:
why change this? return null is correct because we do not want to throw exception

Reply:
here return null is reasonable, I check caller, they have deal with null. Let me restore this behavior

Reply:
done